src/main/docker/Dockerfile

```diff
@@ -71,33 +71,31 @@ RUN dpkg-reconfigure --terse --frontend noninteractive locales
 
 # Set up our configuration files
 COPY docker-entrypoint.sh /usr/local/bin/
-COPY configs/"${project.artifactId}.properties.tmpl" /etc/"${project.artifactId}/${project.artifactId}.properties.tmpl"
-COPY configs/"${project.artifactId}.properties.default" /etc/"${project.artifactId}/${project.artifactId}.properties.default"
-COPY configs/logback.xml /etc/"${project.artifactId}"/logback.xml
+COPY configs/bucketeer.properties.tmpl /etc/bucketeer/bucketeer.properties.tmpl
+COPY configs/bucketeer.properties.default /etc/bucketeer/bucketeer.properties.default
+COPY configs/logback.xml /etc/bucketeer/logback.xml
```

Comment:
Curious, is there a reason to switch to hardcoded files?

Reply:
I think I approached it from the other direction... what's the value of using a parameterized version? If this was going to be a generic Dockerfile that other applications could also use then it might make more sense to use the artifactId, but here what is the value of it? And, it seems to just be less transparent to me (without a benefit).